### ISSUE TITLE
images not scaling

### ISSUE THREAD

**Issue by StephenAtty** [jthingelstad/foreground#345]
### Setup

- Browsers and versions:
- MediaWiki version:  1.31.1
- Database version:  
- PHP version: 7.2.16-1
- Foreground version:  2.2.0
- Extension version (if connected to the issue):

### Issue

images don't resize for mobile devices, By default wikimedia creates them at 640px which is too wide for many mobile devices.  

I might have missed it somewhere but is there any way of making the image size responsive so that the aspect ratio is maintained but the image fits the screen.

Example here:

http://www.pubnight.org.uk/Talk:Steve

**Comment by Hutchy68** [comment]
So one of the things that always bothered me was how images are handled by Foreground. I pushed some changes to another skin recently, I decided to roll them into this skin. 

The main reason you were seeing the problem is you hard set your thumb with a px width. All the changes in the `develop` branch should take care of that issue with a `max-width: 100%;`. Additionally, there are improvements to thumb, captions, enlarge link in caption, bordering, float, nofloat on smaller screens. 

Note, you set your thumb to none, which essentially is `tnone` in the class of the image wrap. Right now, I have it set to center the image which kind of makes sense when I did this for the other skin. A user isn't floating right or left, they are setting no float which essentially is none. Regular insertion of an image is nothing or left-justified. Just using thumb without the `...|right or left|...` automatically sets it as a right float. In otherwords, a left floated thumb is a bit of a unicorn in use but available.

Hope that is all making some sense. Anyway, pull down and use the `v2.2.1-dev` and it should fix the issue and add some more thumbed image enhancements.

Floated right
![image](https://user-images.githubusercontent.com/3527469/54478378-df3f5d00-47e7-11e9-9e78-8444aba2c064.png)

Float none
![image](https://user-images.githubusercontent.com/3527469/54478433-73112900-47e8-11e9-9ff5-fcba1aeeb367.png)

Responsive larger than width of viewport
![image](https://user-images.githubusercontent.com/3527469/54478447-9c31b980-47e8-11e9-8a8d-19dce3cb88ad.png)

Responsive image smaller than width of viewport
![image](https://user-images.githubusercontent.com/3527469/54478470-e155eb80-47e8-11e9-86ef-2a4b71386b31.png)
